src/cascadia/TerminalApp/App.cpp

```diff
@@ -866,7 +885,7 @@ namespace winrt::TerminalApp::implementation
         // Don't capture a strong ref to the tab. If the tab is removed as this
         // is called, we don't really care anymore about handling the event.
         std::weak_ptr<Tab> weakTabPtr = hostingTab;
-        term.TitleChanged([this, weakTabPtr](auto newTitle) {
+        term.TitleChanged([this, weakTabPtr, settings](auto newTitle) {

```

Comment:
I can't figure out what `TitleChanged` and `GotFocus` are doing with the `settings` they capture.